### PR TITLE
docs: warn that README contains newest features

### PR DESCRIPTION
**Pull request by nfischer**
Might be a good idea, in light of #405 and #406

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,14 @@ $ shx touch foo/bar.txt
 $ shx rm -rf foo
 ```
 
+## A quick note about the docs
+
+For documentation on all the latest features, check out our
+[README](https://github.com/shelljs/shelljs). To read docs that are consistent
+with the latest release, check out [the npm
+page](https://www.npmjs.com/package/shelljs) or
+[shelljs.org](http://documentup.com/shelljs/shelljs).
```
Comment (ariporad) [inline README.md]: Actually, we don't own shelljs.org, and documentup is generated from master.

Comment (nfischer) [inline README.md]: Shelljs.org redirects to documentup, right? And documentup is only generated when we actually generate it, which is usually once per release. Npm is pretty reliable though, and there's no way that its docs would be wrong.

Comment (ariporad) [inline README.md]: Hmm... I didn't know it did that. TIL.

**Comment by ariporad** [comment]
LGTM! But could you rebase?

**Comment by nfischer** [comment]
@ariporad If we switch to a `dev` branch like you suggested, it would probably cleaner to just close this. That way docs would not get updated with new features until the next release.

If we find an error with the docs (maybe something is confusing or incomplete), we can merge that into `master` sooner and rebase `dev` off of `master`.

**Comment by ariporad** [comment]
SGTM

**Comment by ariporad** [comment]
@nfischer: One problem with this is that we would push the notice to npm, which wouldn't make much sense.

**Comment by nfischer** [comment]
@ariporad What do you mean by the latest notice? Not quite following you.

**Comment by ariporad** [comment]
If we add a notice that the README is the dev version, then when we push
the README to npm, the notice will be included. Which doesn't make much
sense.


Ari

**Comment by nfischer** [comment]
I see your point. This notice isn't intended to say so much as "these docs are out of date, don't trust them," but more as "regardless of whether you're at Github or npm, the latest features are at Github and the most-recent-release docs are at npm." If there's a way to better phrase that, let's do it.

**Comment by ariporad** [comment]
KK. LGTM!

**Pull request merged**